README.md

```diff
@@ -3,7 +3,7 @@
 >I'm afraid I've run out of free time to work on the Lockbox, so I'm looking for anyone interested in maintaining/growing the project.  If you're interested, open an issue and request to be a collaborator.
 ---
 # The Lua Lockbox
-[![Build Status](https://travis-ci.com/somesocks/lua-lockbox.svg)](https://travis-ci.com/somesocks/lua-lockbox) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Luacheck](https://github.com/somesocks/lua-lockbox/workflows/Luacheck/badge.svg)](https://github.com/somesocks/lua-lockbox/actions)
+[![Build Status](https://travis-ci.com/somesocks/lua-lockbox.svg)](https://travis-ci.com/somesocks/lua-lockbox) [![License][license]](LICENSE) [![Luacheck](https://github.com/somesocks/lua-lockbox/workflows/Luacheck/badge.svg)](https://github.com/somesocks/lua-lockbox/actions)
```

Comment:
I think you can just modify the original file to point to `https://img.shields.io/badge/License-MIT-brightgreen.svg` instead of moving the link reference to the bottom.

Reply:
Right many thanks, addressed